src/helpers/CameraHelper.js

```diff
@@ -169,7 +169,7 @@ class CameraHelper extends LineSegments {
 		// target
 
 		colorAttribute.setXYZ( 38, target.r, target.g, target.b ); colorAttribute.setXYZ( 39, target.r, target.g, target.b ); // c, t
-		colorAttribute.setXYZ( 40, cross.r, cross.g, cross.b ); colorAttribute.setXYZ( 41, cross.r, cross.g, cross.b ); // p, c
+		colorAttribute.setXYZ( 40, target.r, target.g, target.b ); colorAttribute.setXYZ( 41, target.r, target.g, target.b ); // p, c
```

Comment:
Currently, the line from the light position to the center of the far plane is a different color outside the frustum than it is inside the frustum.

 I am not sure there is a compelling reason to change it.